tests/meson.build

```diff
@@ -9,9 +9,13 @@ test_programs = [
   )],
 ]
 
+no_sanitizers = cc.get_supported_arguments(['-fno-sanitize=all'])
+
 executable(
   'try-syscall',
   'try-syscall.c',
+  c_args : no_sanitizers,
+  link_args : no_sanitizers,
```

Comment:
Does this not work?
```meson
  override_options: ['b_sanitize=none'],
```

Reply:
You're right, that's a nicer way to do it.